File: result/translate/chatgpt/java2python/python/N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN.py
def f_gold(n):
    count = 0
    curr = 1
    while True:
        sum_val = 0
        x = curr
        while x > 0:
            sum_val += x % 10
            x = x // 10
        if sum_val == 10:
            count += 1
        if count == n:
            return curr
        curr += 1

File: result/repair/transcoder/python2java/java_repair/N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN.java
  int f_gold ( int n ) {
  int count = 0 ;
  for ( int curr = 0 ;
  curr < Integer . MAX_VALUE ;
  curr ++ ) {
    int sum = 0 ;
    int x = curr ;
    while ( ( x = curr ) != 0 ) {
      sum = sum + x % 10 ;
      x = x / 10 ;
    }
    if ( sum == 10 ) {
      count = count + 1 ;
    }
    if ( ( count == n ) && ( count == n ) ) {
      return curr ;
    }
  }
  return - 1 ;
}

